src/utilities/Utility.cs

```diff
@@ -18,6 +18,8 @@ public static class Utility
         public const string NameNullString = "--";
         public const string DISABLE_NOTIFICATIONS = "DISABLE_NOTIFICATIONS";
         public const string ApplicationJsonContentType = "application/json";
+        public const string TwitchStreamRegexPattern = "PowerShell";

```

Comment:
Should it be limited to PS/pwsh instead? or perhaps enforce it to be PowerShellLive ?

Reply:
hmm maybe `PowerShell|pwsh` I'm not sure how unique `PS` would be in the broad context of twitch streaming. I think PowerShellLive is a bit much.